k8s/hamm/hamm-cost-updater-deployment.yaml.ctmpl

```diff
@@ -44,6 +44,20 @@ spec:
           ports:
             - containerPort: 8080
               name: app-http
+          env:
```

Comment:
We're trying to get away from using env vars this way, and instead put things in `.env` files. This allows us to keep all our secrets in one place--vault--and only worry about making sure the Vault token is up to date, instead of having secrets spread out in lots of places. The POSTGRES_DB_HOST should also go in the `env` file but it can be hardcoded. There are more docs on env files at https://github.com/broadinstitute/dsp-k8s-deploy/blob/master/adding-a-service.md#creating-your-init-container-images